2.10/CHANGELOG-v2.10.rst

```diff
@@ -5217,7 +5222,6 @@ Mongodb
 - community.mongodb.mongodb_shard - Add or remove shards from a MongoDB Cluster
 - community.mongodb.mongodb_shutdown - Cleans up all database resources and then terminates the mongod/mongos process.
 - community.mongodb.mongodb_status - Validates the status of the cluster.
-- community.mongodb.mongodb_stepdown - Step down the MongoDB node from a PRIMARY state.
```

Comment:
Note that this removal is correct, since the module is already listed a bit up. It is not in this specific list as its namespace is correct, as opposed as all the other modules, which end up in the`Community` and then `Mongodb` subsections.

Once ansible-collections/community.mongodb#235 has been fixed, the changelog should look a lot better again.